### ISSUE TITLE
As seen on - content

### ISSUE THREAD

**Issue by Maxhodges** [white-rabbit-japan/wreHomePage#9]
select appropriate logos, including CNN, Akihabara News, Gizmodo etc.
maybe make them all white on green. less busy than color logos.

**Comment by Maxhodges** [comment]
@sabinaFischer OK how about we chanage background ot green here and do the logs all in white like before?

Could you prepare white logos for 
Gizmodo, Akihabara news, Reddit, StumbleUpon

**Comment by SabinaFischer** [comment]
I left them all a little big for now but let me know if these are ok. (might look blank since they will show as white on white?)

![stumbleupon_01](https://cloud.githubusercontent.com/assets/13582674/9003556/f2d20b8a-37a9-11e5-8306-942b903d45fe.png)
![gizmodo_logo](https://cloud.githubusercontent.com/assets/13582674/9003555/f2cfa2a0-37a9-11e5-990a-2037a04682ab.png)
![pl_akihabaranews](https://cloud.githubusercontent.com/assets/13582674/9003557/f2d4fd36-37a9-11e5-90c9-73052958444d.png)
![reddit-logo](https://cloud.githubusercontent.com/assets/13582674/9003558/f2d4f7fa-37a9-11e5-95ec-6cff9e8fe60b.png)

**Comment by SabinaFischer** [comment]
Yeah they look blank... noticed an issue with the reddit one though. Here it is updated 

![reddit-logo](https://cloud.githubusercontent.com/assets/13582674/9003588/262bf3d8-37aa-11e5-876c-3cb7fec0494b.png)
